.travis.yml

```diff
@@ -0,0 +1,24 @@
+dist: xenial
```

Comment:
Can you use a python container to start builds? ([Running python container with geckodriver](http://www.columbia.edu/~njn2118/journal/2016/10/28.html)). This file will be way simpler that way.